models/attribute-group/src/types.ts

```diff
@@ -10,7 +10,7 @@ export type TAttributeGroupGraphql = Omit<
 > & {
   name: string;
   nameAllLocales?: TLocalizedStringGraphql | null;
-  description?: string;
```

Comment:
Many consumers GraphQL queries are expecting `null` instead of undefined so I'm extending these types.